helm-charts/infisical-standalone-postgres/values.yaml

```diff
@@ -13,6 +13,31 @@ infisical:
   # -- Automatically migrates new database schema when deploying
   autoDatabaseSchemaMigration: true
 
+  autoBootstrap:
+    # -- Enable auto-bootstrap of the Infisical instance
+    enabled: false
```

Comment:
i think this should be false to not introduce any new changes? make it more opt in